DigitalExcellence/dex-backend
# Feature/313 get user projects

## Description

For the project profile page we are going to need an endpoint to fetch the projects that belong to the user

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a project on a user
2. Make a request to {dexAPI}/user/projects
3. The API should return the user's projects

## Link to issue
#313 

Closes: #313
